### DISCUSSION TITLE
http://www.andrewheiss.com/blog/2017/08/10/exploring-minards-1812-plot-with-ggplot2/

### DISCUSSION THREAD

**Discussion by andrewheiss** [andrewheiss/ath-quarto#25] [Blog comments]
Exploring Minard’s 1812 plot with ggplot2 | Andrew Heiss 
 http://www.andrewheiss.com/blog/2017/08/10/exploring-minards-1812-plot-with-ggplot2/

**Comment by andrewheiss** [comment]
```header
Originally posted by Thomas Haslam on 2017-08-15 11:19:57
````
Great stuff! Learned a number of new tricks from one post. Thank you.

**Comment by andrewheiss** [comment]
```header
Originally posted by Craig Williams on 2017-08-24 03:27:14
````
There\'s a small typo in the spelling of \'Celsius\'

**Reply by andrewheiss** [reply]
```header
Originally posted on 2017-08-24 05:28:37
````
Ha, that\'s my Americanness showing. Fixed now. Thanks!

**Comment by mikeleadbetter1-creator** [comment]
An inspiration. A chart is just a chart but a great visualisation can change the way we all look at things entirely.
